doskvol-tools/src/components/common/baseTable/BaseTable.vue

```diff
@@ -0,0 +1,94 @@
+<script setup lang="ts">
+
+import type {TableData} from "@/components/common/baseTable/TableData";
+import {ref} from "vue";
+
+const props = defineProps<{
+  tableData: TableData,
+}>();
+
+const emit = defineEmits<{
+  (e: 'chosenTd', value: string)
+}>()
+
+const tableSize = props.tableData.rows[0].values.length;
+
+// function randomInteger(min: number, max: number) {
+//   return Math.floor(Math.random() * (max - min + 1)) + min;
+// }
+//
+// const randomDiceResult = randomInteger(0, b.rows[b.rows.length - 1].diceResult.to);
+// const randomRow = b.rows.find(x => randomDiceResult <= x.diceResult.to);
+// if (randomRow === undefined) throw new Error("HELP");
+// const randomValue = randomRow.values[randomInteger(0, randomRow.values.length - 1)]
+//to będzie logika do random
+//TODO przenieść do odpowiedniego pliku
+const formatDiceResult = (from: number, to: number): string => {
+  if (from === to) return from.toString();
+  return `${from}-${to}`;
+}
+const chosenTd = ref("");
```

Comment:
`chosedTd`? Maybe chosenTableSection would be a better name.

Reply:
Ok, I changed it